pkg/kubernetes/views/imagestream-body.html

```diff
@@ -1,3 +1,14 @@
+<div ng-repeat="statustags in stream.status.tags">
+  <div ng-repeat="condition in statustags.conditions" ng-if='condition.status == "False"'
+      class="alert alert-danger alert-dismissable">
+    <button type="button" class="close" data-dismiss="alert" aria-hidden="true">
+      <span class="pficon pficon-close"></span>
+    </button>
+    <span class="pficon pficon-error-circle-o"></span>
+    <strong>{{ condition.reason }} on {{ condition.type }}:</strong> {{ condition.message }} {{ condition.lastTransitionTime }}
```

Comment:
> Do we really need the reason and the type?

Probably not. Honestly I'm not clear on all the error conditions so I'm not sure how these might possibly be presented.